test/scales/scaleTests.ts

```diff
@@ -8,12 +8,12 @@ describe("Scales", () => {
       return true; // doesn't do anything
     };
     var scale = new Plottable.Scales.Linear();
-    scale.broadcaster.registerListener(null, testCallback);
+    scale.onUpdate(testCallback);
     var scaleCopy = scale.copy();
     assert.deepEqual(scale.domain(), scaleCopy.domain(), "Copied scale has the same domain as the original.");
     assert.deepEqual(scale.range(), scaleCopy.range(), "Copied scale has the same range as the original.");
-    assert.notDeepEqual(scale.broadcaster, scaleCopy.broadcaster,
-                              "Broadcasters are not copied over");
+    assert.notDeepEqual((<any>scale)._callbacks, (<any>scaleCopy)._callbacks,
```

Comment:
Not quite right, since we can't deep-equal test `Set`s. Really we want to ensure that the copied scale has no existing callbacks:
```Typescript
assert.lengthOf((<any> scaleCopy)._callbacks.values(), 0, "copied scale has no callbacks attached");
```
Alternatively, force an update on `scaleCopy` and see if the callbacks get triggered.